src/locales/es.json

```diff
@@ -178,7 +178,7 @@
       "header": "¿Necesita usar su móvil para tomar fotos?",
       "submessage": "Continúe la verificación de forma segura en su dispositivo móvil"
     },
-    "tips": "Importante"
+    "tips": "Consejos"
```

Comment:
The word "Consejos" sounds more like a personal advic, rather than a recommendation. I would use "Recomendaciones"

Reply:
Ah, ok. Will update the text.